modules/serverless-python-template.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-python-functions.adoc
+
+[id="serverless-python-template_{context}"]
+= Python function template structure
+
+When you create a Python function using the `kn func` CLI, the project directory looks similar to a typical Python project. Python functions have very few restrictions. The only requirements are that your project contains a `func.py` file which contains a `main()` function, and a `func.yaml` configuration file.
+
+Both `http` and `event` trigger functions have the same template structure:
+
+.Template structure
+[source,terminal]
+----
+fn
+├── func.py <1>
+├── func.yaml <2>
+├── requirements.txt <3>
+└── test_func.py <4>
+----
+<1> Contains a `main()` function.
+<2> Used to determine the image name and registry.
+<3> Additional dependencies can be added to the `requirements.txt` file as they would be in any other Python project.
+<4> Contains a simple unit test that can be used to test your function locally.
+
+== Dependencies
```

Comment:
Please double-check the rendered assembly to make sure that using this short section is doing what you want it to do. (I might remove the separate heading or turn it into a note.)